lib/static/modules/browser-storage.js

```diff
@@ -0,0 +1,71 @@
+class BrowserStorage {
+    _prefix = 'html-reporter';
+    _deprecatedKeysCollection = [
+        {deprecatedKey: '_gemini-replace-host', newKey: this._getStorageKey('replace-host')}
+    ];
+
+    constructor(storage) {
+        this._storage = storage;
+        this._updateDeprecatedKeys();
+    }
+
+    /**
+     * Format Storage key
+     * @param key
+     * @returns {string}
+     * @private
+     */
+    _getStorageKey(key) {
+        return `${this._prefix}:${key}`;
+    }
+
+    /**
+     *  Helping to migrate on new storage keys
+     *  @private
+     */
+    _updateDeprecatedKeys() {
+        this._deprecatedKeysCollection.forEach(({deprecatedKey, newKey}) => {
+            if (this._storage.hasOwnProperty(deprecatedKey)) {
+                this._storage.setItem(newKey, this._storage.getItem(deprecatedKey));
+                this._storage.removeItem(deprecatedKey);
+            }
+        });
+    }
+
+    /**
+     *  Wrap Storage#setItem method
+     *  Support value serialization
```

Comment:
You have extra spaces before `Wrap` & `Support`. And below in `getItem` description – the same thing.